### ISSUE TITLE
Configurable logging level

### ISSUE THREAD

**Issue by DanPerovich** [bpaauwe/RokuPoly#3]
It would be great if we could configure the logging level using the configuration parameters.

**Comment by bpaauwe** [comment]
Agree. There has been some discussions as part of the PG3 development around this.  I'm waiting to see if PG3 builds in the log level setting to the Polyglot interface before attempting to create another method to set the log level.

Ideally, this is something that should be consistent for all node servers.
